ecosystem/sep-0006.md

```diff
@@ -1010,6 +1011,39 @@ Name | Type | Description
 * `too_large` -- deposit/withdrawal size exceeded `max_amount`.
 * `error` -- catch-all for any error not enumerated above.
 
+### Refunds Object Schema
+
+Name | Type | Description
+-----|------|------------
+`amount_refunded` | string | The total amount refunded to the user, in units of `amount_in_asset`. If `type` is `full`, this amount must match `amount_in`.
+`amount_fee` | string | The total amount charged in fees for processing all refund payments. The sum of all `fee` values in the `payments` object list should equal this value.
+`payments` | array | A list of objects containing information on the individual payments made back to the user as refunds. The schema for these objects is defined in the section below.
+
```

Comment:
Are we assuming the same `asset_code` of the original transaction?

Reply:
We're assuming all refund amounts are in units of `amount_in_asset`

Reply:
Can we make this explicit in the doc?